### PR TITLE
memberships: user page

### PR DESCRIPTION
**Pull request by DTabidze**

### CODE REVIEW

File: core/auth/memberships/index.html [hunk 1]
```diff
@@ -15,6 +16,9 @@ <h1>Groups Managment</h1>
         <input type="text" id="description" name="description">
         <button type="submit">Create Group</button>
     </form>
+    {{ else }}
+    <h1>User: {{ .CurrentUser }}</h1>
```
Comment (giolekva) [inline core/auth/memberships/index.html:20]: Let's always render this, even when logged in user is looking at their own page. So lets move it before line 10

File: core/auth/memberships/index.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="/static/pico.2.0.6.min.css">
 </head>
 <body class="container">
+    {{ if .LoggedInUserPage }}
     <h1>Groups Managment</h1>
```
Comment (giolekva) [inline core/auth/memberships/index.html:11]: maybe let's remove this header?

**Review by giolekva — CHANGES_REQUESTED** [review]
nice, looks good. Left only couple minor comments

File: core/auth/memberships/index.html (hunk 1 above)
Comment (DTabidze) [reply core/auth/memberships/index.html]: Done.

File: core/auth/memberships/index.html (hunk 2 above)
Comment (DTabidze) [reply core/auth/memberships/index.html]: Done.

**Review by giolekva — APPROVED** [review]

**Pull request merged**